lib/smartsheet/client.ex

```diff
@@ -1,9 +1,26 @@
 defmodule Smartsheet.Client do
+  alias Smartsheet.ClientBehaviour
+
   use HTTPoison.Base
 
   @smartsheet_api_version "2.0"
   @base_url "https://api.smartsheet.com"
 
+  @behaviour ClientBehaviour
+
+  @impl ClientBehaviour
+  defdelegate get_sheet(sheet_id, options \\ []), to: Smartsheet.Sheets, as: :get
+
+  @impl ClientBehaviour
+  defdelegate create_sheet(attributes), to: Smartsheet.Sheets, as: :create
+
+  @impl ClientBehaviour
+  defdelegate add_rows(sheet_id, rows), to: Smartsheet.Rows, as: :add_to_sheet
+
+  @impl ClientBehaviour
+  defdelegate update_rows(sheet_id, rows), to: Smartsheet.Rows, as: :update
+
+  @impl HTTPoison.Base
```

Comment:
is it normal / ok that i have two different things i'm `@impl` here? `ClientBehaviour` && `HTTPoison.Base` ?

Reply:
That should be fine afaik.